docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,182 @@
-Ubuntu specific instructions
\ No newline at end of file
+# Installing LORIS In-Depth
+
+This guide assumes an installation on an Ubuntu server. 
+
+If you are using CentOS, please visit the CentOS installation guide for 
+instructions on installation.
+
+We also have instructions for how to set up LORIS for macOS. However, only
+ Ubuntu and CentOS are officially supported by the team so we may be unable to
+ help you with issues arising on other systems.
+
+
+When you've completed this guide, further set up is required. Please visit
+[the Setup page](https://github.com/aces/Loris/wiki/Setup) in order to be
+able to use LORIS properly.
+
+## Prequisities
+
+### Server Stack
+
+LORIS requires a LAMP stack in order to run, specifically:
+* Apache 2.4
+* MySQL 5.7 (or MariaDB 10.1) (or higher)
+* PHP 7.3 (or higher)
+
+Additionally, LORIS requires the following package managers:
+* npm 8.0 (or higher)
+* composer
+
+This guide does not cover installation of these requirements.
+
+### Apt Packages
+The following Ubuntu packages are required and should be installed using 
+`sudo apt install ...`.
+
+* git
+* curl
+* wget
+* zip
+* unzip
+* php-json
+* make
+* software-properties-common
+* php7.3-mysql
+* php7.3-xml
+* php7.3-json
+* php7.3-mbstring
+* php7.3-gd
+* libapache2-mod-php7.3
+
+### Database set up
+
+MySQL (or MariaDB) must be installed and a root or admin-level MySQL user must
+be created before continuing. (This is not the same as a unix root credential.)
+
+## Getting the source code
+
+### LORIS for Development
+If installing LORIS for development purposes, clone the source code from our
+GitHub repository to the web root. Then change to this directory.
+
+Example command assuming LORIS will be served from `/var/www/`:
+
+```bash
+git clone git@github.com:aces/Loris.git /var/www/loris
+cd /var/www/loris/
+```
+
```

Comment:
add a note here something like: when documentation refers to the Loris root directory, it means `/var/www/loris`/

Reply:
I don't use that term in that way in this document and it's also not quite right. The LORIS root directory is the folder where all the LORIS files are contained. `/var/www/loris/` is one example name and path to a LORIS root directory. If someone installed LORIS at `/home/john/my_coding_projects/TestLoris` then 'TestLoris' is the LORIS root directory.

I also don't want this document to function as a glossary for example values for configuration settings. If that's useful we can create a separate document for that which other docs can link to. Otherwise we would have to duplicate this information across CentOS and macOS install guides, at least.